File: leetcode/2798-number-of-employees-who-met-the-target/numberOfEmployeesWhoMetTarget.ts
function numberOfEmployeesWhoMetTarget(hours: number[], target: number): number {
  let number = 0;

  for (const hour of hours) {
    if (hour < target) {
      continue;
    }

    number++;
  }


  return number;
};

export default numberOfEmployeesWhoMetTarget;

File: leetcode/2798-number-of-employees-who-met-the-target/numberOfEmployeesWhoMetTarget.rs
use crate::Solution;

impl Solution {
    pub fn number_of_employees_who_met_target(hours: Vec<i32>, target: i32) -> i32 {
        let mut number = 0;

        for hour in hours {
            if hour < target {
                continue;
            }

            number += 1;
        }

        return number;
    }
}
